packages/ckeditor5-package-generator/lib/templates/ts/package.json

```diff
@@ -48,9 +48,13 @@
     "@ckeditor/ckeditor5-theme-lark": ">=<%= packageVersions.ckeditor5 %>",
     "@ckeditor/ckeditor5-upload": ">=<%= packageVersions.ckeditor5 %>",
     "@types/chai": "^4.3.1",
+    "@types/ckeditor__ckeditor5-autoformat": "^31.0.1",
+    "@types/ckeditor__ckeditor5-basic-styles": "^28.0.2",
+    "@types/ckeditor__ckeditor5-block-quote": "^29.0.4",
     "@types/ckeditor__ckeditor5-core": "^33.0.3",
     "@types/ckeditor__ckeditor5-editor-classic": "^27.1.2",
     "@types/ckeditor__ckeditor5-essentials": "^28.0.3",
+    "@types/ckeditor__ckeditor5-list": "^32.0.1",
     "@types/ckeditor__ckeditor5-ui": "^32.0.2",
```

Comment:
The list of imported community types is not complete here. In the sample file you use also `ckeditor5-basic-styles`, `ckeditor5-heading`, `ckeditor5-image`, `ckeditor5-upload`, `ckeditor5-indent`, `ckeditor5-link`, `ckeditor5-media-embed`, `ckeditor5-paragraph`, `ckeditor5-table` and `ckeditor5-code-block`.

There is no "missing type" error reported here, because these types are indirectly installed in `node_modules` in the generated TypeScript package by the `@types/ckeditor__ckeditor5-core` package, but I think we should define them explicitly if we need them.